src/lib/connections.ts

```diff
@@ -11,12 +11,21 @@ import type {SignerPayloadRaw, SignerResult} from "@polkadot/types/types";
 import type {SubmittableExtrinsic} from "@polkadot/api/promise/types";
 import type {EventRecord, ExtrinsicStatus} from "@polkadot/types/interfaces";
 
-export let providers = {
+type ProviderMap = {
```

Comment:
Maybe this should be an interface instead of a type? This was a copilot suggestion, I was trying to correct TS errors.

Reply:
Have you tried: 
```typescript
export ProviderMap: Record<string,string> = {
    Rococo: 'wss://rpc.rococo.frequency.xyz',
    Localhost: 'ws://localhost:9944',
    Other: 'wss://some.node',
}
```
then just use ProviderMap instead of `providers` ?(https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type)

Reply:
Cool, this works.